lib/vdsm/storage/fileVolume.py

```diff
@@ -528,9 +529,19 @@ def _create_cow_volume(
             cls.log.info("Request to create COW volume %s with capacity = %s",
                          vol_path, capacity)
 
+            vol_format = sc.fmt2str(sc.COW_FORMAT)
```

Comment:
vol_format is usually used for sc.{COW,RAW}_FORMAT. "format" would be a better name.

And we should also use format in the caller to _run_create_image().